README.md

```diff
@@ -145,9 +145,6 @@ and the language:
     -   Operator overloading
     -   Lexical and syntactic structure
     -   Code organization and modular structure
--   A prototype interpreter demo that can both run isolated examples and gives a
-    detailed analysis of the specific semantic model and abstract machine of
-    Carbon. We call this the [Carbon Explorer](/explorer/).
```

Comment:
I wonder if this could stay but link to the archived repo? WDYT?

Reply:
The way I read this, the paragraph is *extremely* promotional of the explorer ("detailed analysis", "abstract machine of Carbon"). Does that match your view of what the explorer is today?

Reply:
Or more so, does the way this fits in match the current project status?

Reply:
Works for me. Suggested an edit but I'm fine with it as-is if you'd prefer to devote fewer words to explorer.